src/app/error-boundary/ErrorBoundary.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router';
+import { Page404, Page500 } from '../../pages/errors';
+import styles from './ErrorBoundary.css';
+
+class ErrorBoundary extends Component {
+    static getDerivedStateFromProps(props, state) {
+        if (props.location.pathname !== state.pathname) {
+            return {
+                hasError: false,
+                error: undefined,
+                pathname: props.location.pathname,
+            };
+        }
+
+        return null;
+    }
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            hasError: false,
```

Comment:
I think we do not this need, we can just check against `error`.

Reply:
I've removed it, thanks 👍